distribution/docs/src/main/resources/_contents/quickstartcontents.adoc

```diff
@@ -0,0 +1,174 @@
+
+=== Getting Started
+
+This quick tutorial will enable install, configuring and using a basic instance of {branding}.
+
+These steps will demonstrate:
+
+- [*] Prerequisites
+- [*] Installation
+- [*] Catalog Capabilities: Ingest and query using every endpoint
+- [*] Use of the Content Framework
+- [*] Metrics Reporting
+
+==== Prerequisites
+
+* Install/Upgrade to Java 8 http://www.oracle.com/technetwork/java/javase/downloads/index.html[J2SE 8 SDK]
+* Supported platforms are *NIX - Unix/Linux/OSX, Solaris, and Windows.
+* http://www.oracle.com/technetwork/java/javase/downloads/index.html[JDK8] must be installed.
+* The `JAVA_HOME` environment variable must be set to the location where the JDK is installed.
+
+.Example of Setting Up on *NIX
+----
+JAVA_HOME=/usr/java/jdk1.8.0
+export JAVA_HOME
+----
+
+.Example of Setting Up JAVA_HOME on Windows
+----
+set JAVA_HOME=C:\Program Files\Java\jdk1.8.0
+----
+
+.*NIX
+[WARNING]
+====
+Unlink `/usr/bin/java` if it is already linked to a previous version of the JRE:
+`unlink /usr/bin/java`
+====
+
+[TIP]
+====
+Verify that the JAVA_HOME was set correctly.
+
+.*NIX
+----
+echo $JAVA_HOME
+----
+
+.Windows
+----
+echo %JAVA_HOME%
+----
+====
+
+* {branding} installation zip file.
+* A web browser.
+* For Linux systems, increase the file descriptor limit by editing `/etc/sysctl.conf` to include or change the following:
+
+.File Descriptor Limit
+----
+fs.file-max = 6815744
+----
+
+.Restart
+[WARNING]
+====
+For the change to take effect, a restart is required.
+====
+
+.Restart Command
+----
+init 6
+----
+
+==== Installing {branding}
+
+. Install {branding} by unzipping the https://github.com/codice/ddf/releases/latest[zip file].
+. This will create an installation directory, which is typically created with the name and version of the application.
+This installation directory will be referred to as `<DISTRIBUTION_INSTALL_DIR>`.
+(Substitute the actual directory name.)
+. Start {branding} by running the `<DISTRIBUTION_INSTALL_DIR>/bin/ddf` script (or `ddf.bat` on Windows).
+
+==== Configuring
+
+.. Go to https://localhost:8993/admin.
+.. Enter the default username of "admin" (no quotes) and the password of "admin" (no quotes).
+.. Follow the prompts installer prompts for a standard installation.
+... Click start to begin the setup process.
+... Configure guest claims attributes or use defaults.
+... Select Standard Installation.
+... On the System Configuration page, configure any port or protocal changes desired and add any keystores/truststores needed.
+... Click *Finish*
+
+==== Catalog Capabilities
+
+. Create an entry in the Catalog by ingesting a valid https://codice.atlassian.net/wiki/download/attachments/1179756/geojson_valid.json?version=1&modificationDate=1368249436010&api=v2[GeoJson file].
+// Should this be rewritten to use Search UI to ingest? or should that be added?This ingest can be performed using:
```

Comment:
I'll move these other instructions to the longer "Running DDF" section of the Managing page.